playbooks/common/openshift-cluster/upgrades/pre/verify_memory_and_diskspace.yml

```diff
@@ -0,0 +1,12 @@
+- name: Verify Host Requirements
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: "pre-upgrade"
```

Comment:
So if this is being included in an actual upgrade playbook above, I would say that should be just "upgrade" context. If you want to reuse the playbook it should probably not specify a context, instead relying on including playbooks to set context.

Also I don't like calling this `verify_memory_and_diskspace.yml` -- I assume we're going to add more checks to the same play in later iterations.